debezium-core/src/main/java/io/debezium/transforms/ExtractNewRecordStateConfigDefinition.java

```diff
@@ -67,6 +67,71 @@ public static DeleteHandling parse(String value, String defaultValue) {
         }
     }
 
+    public enum DeleteTombstoneHandling implements EnumeratedValue {
+        DROP("drop"),
+        TOMBSTONE("tombstone"),
+        REWRITE("rewrite"),
+        REWRITE_WITH_TOMBSTONE("rewrite-with-tombstone");
+
+        private final String value;
+
+        DeleteTombstoneHandling(String value) {
+            this.value = value;
+        }
+
+        @Override
+        public String getValue() {
+            return value;
+        }
+
+        /**
+         * Determine if the supplied value is one of the predefined options.
+         *
+         * @param value the configuration property value; may not be null
+         * @return the matching option, or null if no match is found
+         */
+        public static DeleteTombstoneHandling parse(String value) {
+            if (value == null) {
+                return null;
+            }
+            value = value.trim();
+            for (DeleteTombstoneHandling option : DeleteTombstoneHandling.values()) {
+                if (option.getValue().equalsIgnoreCase(value)) {
+                    return option;
+                }
+            }
+            return null;
+        }
+
+        /**
+         * Determine if the supplied value is one of the predefined options.
+         *
+         * @param value the configuration property value; may not be null
+         * @param defaultValue the default value; may be null
+         * @return the matching option, or null if no match is found and the non-null default is invalid
+         */
+        public static DeleteTombstoneHandling parse(String value, String defaultValue) {
+            DeleteTombstoneHandling mode = parse(value);
+            if (mode == null && defaultValue != null) {
+                mode = parse(defaultValue);
+            }
+            return mode;
+        }
+    }
+
+    public static final Field HANDLE_TOMBSTONE_DELETES = Field.create("delete.tombstone.handling.mode")
+            .withDisplayName("Handle delete records")
+            .withEnum(DeleteTombstoneHandling.class)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.MEDIUM)
+            .optional()
+            .withDescription("How to handle delete records. Options are: "
+                    + "drop - will remove both delete and tombstone,"
+                    + "tombstone - will convert delete to tombstone and remove tombstone (the default),"
```

Comment:
```suggestion
                    + "tombstone - will only leave a tombstone in the stream (the default),"
```
I am not happy with that explanation but from a user's perspective I feel that explains more what the user will get (instead of technically describing that the delete event is converted into a tombstone and the original tombstone is dropped from the stream).
wdyt?